Casks/l/labplot.rb

```diff
@@ -0,0 +1,24 @@
+cask "labplot" do
+  arch arm: "arm64", intel: "x86_64"
+
+  version "2.11.1"
+  sha256 arm:   "61af87d94f83f9e254b2ada3ae801c9fdc9c034047c13aa7dd10dfc24b846380",
+         intel: "59180a91db2661dc1f0bd52a684213b8ea8f449be3fe108636f96e1438411650"
+
+  url "https://download.kde.org/stable/labplot/labplot-#{version}-#{arch}.dmg"
+  name "labplot"
+  desc "Free, open source and cross-platform Data Visualization and Analysis software"
+  homepage "https://labplot.kde.org/"
+
+  livecheck do
+    url "https://download.kde.org/stable/labplot/"
+    regex(/href=.*?labplot[._-]?(\d+(?:[.-]\d+)+)-#{arch}.dmg/i)
+    strategy :page_match
```

Comment:
```suggestion
    url "https://download.kde.org/stable/labplot/"
    regex(/href=.*?labplot[._-]v?(\d+(?:\.\d+)+)[._-]#{arch}.dmg/i)
```

We can omit `strategy :page_match` since the `PageMatch` strategy is used as default here.